CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
   view.
 - Add a "Users" view to the Team projects area, to view other users on your
   team, with their conversion project counts.
+- Add a "by user" view to the Team projects area, where a user can view projects
```

Comment:
Honestly I wouldn't worry about this - I see no value in preventing this until we find a reason?

Reply:
Do you want me to drop this whole commit or just the permissions part?